src/components/CountryCard.jsx

```diff
@@ -0,0 +1,27 @@
+import { BsArrowRightCircle } from 'react-icons/bs';
+import '../styles/CountryCard.css';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+export default function CountryCard({ flags, name }) {
+  return (
+    <div className="country-card">
+      <Link to={`/${name.common.toLowerCase()}`}>
+        <img src={flags.svg} alt={`${name.common} falg`} />
+        <div className="overlay-container">
+          <BsArrowRightCircle className="forward-icon" />
+          <h3>{name.common}</h3>
+        </div>
+      </Link>
+    </div>
+  );
```

Comment:
[REQUIRED] Kindly increase the height of the country cards so that they can look more like Nelson's design. It is a requirement for your design to look like the template

[REQUIRED] I can also see some space between the cards in your design. Kindly fix this as there aren't spaces between the cards in the template



|Your Design | Template|
|-------------|--------|
|![image](https://github.com/UbdaNam/africa-covid-19-tracker/assets/62938735/2df8fcf0-7eec-43ca-94d7-08f4389102fa)| ![image](https://github.com/UbdaNam/africa-covid-19-tracker/assets/62938735/046f2610-a8dd-4037-b5b2-0e294e18bb78)|